actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -199,6 +199,23 @@ def headers
       @headers ||= Http::Headers.new(self)
     end
 
+    # Early Hints is an H2 status code that indicates hints to help a client start
```

Comment:
Is H2 an accepted short hand for HTTP2? Otherwise we should probably stick to HTTP2 through out.

Reply:
While it is an accepted shorthand I've replaced this with HTTP/2 because documentation should not assume knowledge of that.